sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_prefetch.cpp

```diff
@@ -16,11 +16,5 @@
 
 // -ffp-model=precise is added to not depend on compiler defaults.
 
-// This test will be adjusted to run on the CPU
-// XFAIL:cpu
```

Comment:
Is the test passing on CPU as well?

Reply:
Yes, that is a separate issue but the test is also xpasssing on the CPU